### DISCUSSION TITLE
Multi-Tenancy App

### DISCUSSION THREAD

**Discussion by urumat** [chatwoot/chatwoot#4112] [Feature Requests]
Hello, 
I would like to know if it would be possible to have a multi-tenancy installation of Chatwoot.

Something like Organization1 would be accessible through org1.chatwoot.com, Organization2 would be accessible through org2.chatwoot.com and so on.

Thanks in advance.

**Comment by jordan-brough** [comment]
We'd be interested in this, and possibly in contributing as well. We're currently using the [ros-apartment](https://github.com/rails-on-services/apartment) gem for multi-tenancy in our core application.

**Comment by sojan-official** [comment]
Chatwoot supports multi-tenancy out of the box.  We use the `tenant identifier` column `account_id` so that the data from any given tenant can be selectively retrieved. 

ref: https://docs.microsoft.com/en-us/azure/azure-sql/database/saas-tenancy-app-design-patterns#f-multi-tenant-app-with-a-single-multi-tenant-database

All tenants access their data from a single root domain. We don't support subdomain based isolation at the moment.
